### PR TITLE
Add style to login and redirect to login page.

### PR DESCRIPTION
**Pull request by anshika29**

### CODE REVIEW

File: components/coins-status/index.js [hunk 1]
```diff
@@ -82,10 +82,17 @@ const CoinsStatus = () => {
 
   return (
     <div className={styles.coinsContainer}>
-      {isError && <p>Something went wrong ...</p>}
-      {isError && isLoginMessage && (
+      {isError && <p>Something went wrong ...&nbsp;</p>}
+      {isError && isLoginError && (
         <p>
-          Please <a href="LOGIN_URL">login to view your balance.</a>
+          Please&nbsp;
```
Comment (Rucha1499) [inline components/coins-status/index.js:88]: Can we have `&nbsp` on next line and properly spaced? Code will look clean. 😃

File: components/coins-status/index.js [hunk 2]
```diff
@@ -82,10 +82,17 @@ const CoinsStatus = () => {
...
+          <a
+            className={styles.loginMessage}
+            href="https://www.realdevsquad.com/"
+          >
+            login
+          </a>
+          &nbsp;to view your balance.
```
Comment (Rucha1499) [inline components/coins-status/index.js:95]: Refer the above review!

**Review by Rucha1499 — CHANGES_REQUESTED** [review]
Thanks for this fix @anshika29!! 🎉 

Adding some comments. kindly check them!

Thank you.

File: components/coins-status/index.js (hunk 1 above)
Comment (swarajpure) [reply components/coins-status/index.js:88]: Not necessarily on the next line, but at least separated by a space maybe? Looking too cluttered now

**Review by Rucha1499 — APPROVED** [review]
Looks good to me! @anshika29 😃

**Pull request merged**